sdk/storage/AzureStorageBlob/Source/Options/ClientMethodOptions.swift

```diff
@@ -72,27 +76,198 @@ public struct ListContainersOptions: RequestOptions {
     ///   - prefix: Return only containers whose names begin with the specified prefix.
     ///   - include: One or more datasets to include in the response.
     ///   - maxResults: Maximum number of containers to return, up to 5000.
-    ///   - timeoutInSeconds: Request timeout in seconds.
+    ///   - timeout: Request timeout in seconds.
     public init(
         clientRequestId: String? = nil,
         cancellationToken: CancellationToken? = nil,
         dispatchQueue: DispatchQueue? = nil,
         prefix: String? = nil,
         include: [ListContainersInclude]? = nil,
         maxResults: Int? = nil,
-        timeoutInSeconds: Int? = nil
+        timeout: TimeInterval? = nil
     ) {
         self.clientRequestId = clientRequestId
         self.cancellationToken = cancellationToken
         self.dispatchQueue = dispatchQueue
         self.prefix = prefix
         self.include = include
         self.maxResults = maxResults
-        self.timeoutInSeconds = timeoutInSeconds
+        self.timeout = timeout
+    }
+}
+
+/// User-configurable options for the `ContainersOperations.delete` operation.
+public struct DeleteContainerOptions: RequestOptions {
+    /// A client-generated, opaque value with 1KB character limit that is recorded in analytics logs.
+    public let clientRequestId: String?
+
+    /// A token used to make a best-effort attempt at canceling a request.
+    public let cancellationToken: CancellationToken?
+
+    /// A dispatch queue on which to call the completion handler. Defaults to `DispatchQueue.main`.
+    public let dispatchQueue: DispatchQueue?
+
+    /// A `PipelineContext` object to associate with the request.
+    public var context: PipelineContext?
```

Comment:
I notice we never take a Context object as a parameter to the initializer for any of these Options types. Was that an intentional decision?

Reply:
Nope! This just wasn't updated after context was added to the RequestOptions. Good catch! Has been fixed.